Makefile

```diff
@@ -1035,3 +1038,17 @@ patch-remote-controller: hack/scripts/patch-remote/.ssh/key_$(PATCH_HOST_IP)_$(P
 		--private-key-file hack/scripts/patch-remote/.ssh/key_$(PATCH_HOST_IP)_$(PATCH_USERNAME) \
 		--config hack/scripts/patch-remote/patch_env.yml \
 		--targets controller
+
+.PHONY: patch-remote-dlx
+patch-remote-dlx: hack/scripts/patch-remote/.ssh/key_$(PATCH_HOST_IP)_$(PATCH_USERNAME)
+	./hack/scripts/patch-remote/patch_remote.py \
+		--private-key-file hack/scripts/patch-remote/.ssh/key_$(PATCH_HOST_IP)_$(PATCH_USERNAME) \
+		--config hack/scripts/patch-remote/patch_env.yml \
+		--targets dlx
+
+.PHONY: patch-remote-autoscaler
+patch-remote-autoscaler: hack/scripts/patch-remote/.ssh/key_$(PATCH_HOST_IP)_$(PATCH_USERNAME)
+	./hack/scripts/patch-remote/patch_remote.py \
+		--private-key-file hack/scripts/patch-remote/.ssh/key_$(PATCH_HOST_IP)_$(PATCH_USERNAME) \
+		--config hack/scripts/patch-remote/patch_env.yml \
+		--targets autoscaler
```

Comment:
Honestly I think we can remove all of the specific patch rules (including dashboard and controller), and only use the generic `patch-remote-nuclio`, relying on the targets specified in `patch_env.yml`.

If there will be changes in the future, It will be easier to maintain one rule instead of 5 duplicates.